omod/src/main/java/org/openmrs/module/queue/web/QueueEntryTransitionRestController.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.queue.web;
+
+import java.util.Date;
+import java.util.Map;
+import java.util.Optional;
+
+import org.openmrs.Concept;
+import org.openmrs.api.APIException;
+import org.openmrs.module.queue.api.QueueServicesWrapper;
+import org.openmrs.module.queue.model.Queue;
+import org.openmrs.module.queue.model.QueueEntry;
+import org.openmrs.module.queue.model.QueueEntryTransition;
+import org.openmrs.module.webservices.rest.web.ConversionUtil;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.BaseRestController;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/**
+ * The main controller that exposes additional end points for order entry
+ */
+@Controller
+@RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + "/queue-entry-transition")
```

Comment:
I think we generally try to namespace the REST stuff by moduleId:

```suggestion
@RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + "/queue/queue-entry-transition")
```

Reply:
Yes, 100% @ibacher , but in this case we already have a resource at `/queue`, it is what returns all of the queues, and specific resources at `/queue/<uuid>` which returns a specific queue.  So that would seem to prevent us from moving to a `queue` namespace unfortunately.  For now, what I've done is consistent with the rest of the module that exists, and I'm trying to maintain a pattern of having a `queue` prefix on all endpoints, which isn't perfect but it's better than nothing.  Do you have any other suggestions?  We could certainly fix this in the future in a major release.

Reply:
I see. Let's save it for a major release then.